engine/api/pom.xml

```diff
@@ -38,7 +38,7 @@
     <dependency>
       <groupId>org.apache.cxf</groupId>
       <artifactId>cxf-bundle-jaxrs</artifactId>
-      <version>2.7.13</version>
+      <version>2.7.18</version>
```

Comment:
Sure.

Wondering if we can remove many of the engine/{projects}; not sure where it is used or who uses that. Looks like a WIP attempt to creating a RESTful api server.